scripts/zones/Windurst_Walls/Zone.lua

```diff
@@ -19,8 +19,14 @@ end
 function onZoneIn(player, prevZone)
     local cs = -1
 
-    if ENABLE_ASA == 1 and player:getCurrentMission(ASA) == tpz.mission.id.asa.A_SHANTOTTO_ASCENSION
-        and (prevZone == tpz.zone.WINDURST_WATERS or prevZone == tpz.zone.WINDURST_WOODS) and player:getMainLvl()>=10 then
+    if ENABLE_ROV == 1 and player:getCurrentMission(ROV) == tpz.mission.id.rov.RHAPSODIES_OF_VANADIEL and player:getMainLvl()>=3 then
+        cs = 30035
+    elseif player:getCurrentMission(ROV) == tpz.mission.id.rov.FATES_CALL and player:getFameLevel(player:getNation()) >= 5 then
```

Comment:
I believe this should be checking `player:getCurrentMission(player:getNation()) > 15` and not `player:getFameLevel(player:getNation()) >= 5`.